lib/types.ts

```diff
@@ -4,6 +4,11 @@ export interface StaticAnalysisOptions {
   distroless: boolean;
 }
 
+export enum ImageTransport {
```

Comment:
what does image transport mean?

Reply:
You can obtain an image from a variety of sources, e.g. container registry, docker socket, filesystem path, etc. I think `skopeo` is calling it "repository type":
https://github.com/containers/skopeo
> Skopeo operates on the following image and repository types:

Maybe it's not a bad idea to use that terminology 😄